### PR TITLE
Space Travelers' Hub

### PR DESCRIPTION
**Pull request by AxelSoler**
# Team project @rega1237 and @AxelSoler 

## The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.

- React App, [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
- Working with the real live data from the SpaceX API
- Routes and view components
- Create directories for all Redux state slice files
- Use useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes. 
- Render UI: conditional components rendering
- Use Jest for testing

All feedback is well received

### CODE REVIEW

File: src/test/rocketsComponent.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Rockets from '../components/Rockets/Rockets';
+
+describe('Rockets', () => {
+  it('Renders correctly', () => {
+    const tree = renderer
+      .create(
+        <MemoryRouter>
+          <Provider store={store}>
+            <Rockets />
+          </Provider>
```
Comment (andersonlebon) [inline src/test/rocketsComponent.test.js:14]: image

- [ ] KIndly note that this Rockets component has some props validation which means some of them are required so kindly make sure you add them when testing this component. Hint :point_down: example

```test.js
escribe('Rockets', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Provider store={store}>
            <Rockets
              id={1}
              name="Rocket 1"
              img=""
              description="description"
              reserved
            />
          </Provider>
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

```

#### Here are your props validations

image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,62 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Give a ⭐️ if you like this project!
 
-### Analyzing the Bundle Size
+## Acknowledgments
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (andersonlebon) [inline README.md:58]: - [ ] To keep your `README` file more professional kindly customize this section with your own information.

**Review by andersonlebon — COMMENTED** [review]
## Hi @AxelSoler  and @rega1237 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/rocketsComponent.test.js (hunk 1 above)
Comment (rega1237) [reply src/test/rocketsComponent.test.js]: Thanks for the review 👍🏻  you are right I forgot to add the props to the component test

**Review by elyor-doniyorov — APPROVED** [review]
Hi @AxelSoler and @rega1237, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Descriptive Readme file
✅ Tests are passing

## Optional suggestions
- [OPTIONAL] In order to make your codebase cleaner kindly delete all boilerplate that created by the CRA(texts, images, styles) from the `public` folder.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**